qemu/tests/balloon_check.py

```diff
@@ -423,7 +424,10 @@ def run(test, params, env):
         else:
             balloon_type = params_tag['balloon_type']
             min_sz, max_sz = balloon_test.get_memory_boundary(balloon_type)
-            expect_mem = int(random.uniform(min_sz, max_sz))
+            if params_tag.get('small_value_check', 'no') == 'yes':
+                expect_mem = int(min_sz)
+            else:
```

Comment:
```
expect_mem = int(random.uniform(min_sz, max_sz))
if params_tag.get('small_value_check', 'no') == 'yes':
     expect_mem = int(min_sz)
```
It will save one line :)

Reply:
Modified.
Thanks

Reply:
expect_mem = int(min_sz) if params_tag.get('small_value_check', 'no') == 'yes' else int(random.uniform(min_sz, max_sz))

Reply:
This could save more lines. However both are ok to me:)